app/services/grading_scale_service.rb

```diff
@@ -14,7 +14,14 @@ def count_for(grading_scale)
     if grading_scale.not_graded
       @term_registrations.ungraded.length
     else
-      range = (grading_scale.min_points..grading_scale.max_points)
+      if grading_scale.grade == "1"
+        range = (grading_scale.min_points..Float::INFINITY)
```

Comment:
Please use an open range instead.

```suggestion
        range = (grading_scale.min_points..)
```